package.json

```diff
@@ -15,7 +15,7 @@
     "copy-landing-app": "cp -rf packages/landing-app/dist/* docs/.",
     "deploy": "gh-pages -d docs -b master",
     "dev:all": "lerna run dev --parallel --scope '!@blueprintjs/{landing-app,table-dev-app}'",
-    "dev:core": "lerna run dev --parallel --scope '@blueprintjs/{core,docs-app}'",
+    "dev:core": "lerna run dev --parallel --scope '@blueprintjs/{core,icons,docs-app}'",
```

Comment:
this change is unnecessary. we rarely work on icons source. you can run `lerna run dev --parallel --scope '@blueprintjs/icons'` in a separate console.

Reply:
in my experience, this change has been very helpful. especially cuz data files come from `icons` package but component comes from `core`. all other dev aliases include package deps, so what's the harm here?